sphinx_immaterial/apidoc/format_signatures.py

```diff
@@ -58,7 +58,7 @@ class CollectSignaturesTransform(sphinx.transforms.SphinxTransform):
 
     def apply(self, **kwargs: Any) -> None:
         collected_signatures = _get_collected_signatures(self.env)
-        for node in self.document.traverse(sphinx.addnodes.desc_signature):
+        for node in self.document.findall(sphinx.addnodes.desc_signature):
```

Comment:
Is this to satisfy a deprecation?

Reply:
Yes

Reply:
I suppose this won't impact us since we're pinned to whatever ver of docutils that sphinx installs. IIRC, that deprecation has been around since docutils v0.16, and sphinx v4+ never pinned to an earlier version of docutils.